dpe/src/commands/derive_child.rs

```diff
@@ -400,52 +406,68 @@ mod tests {
         };
         let mut dpe = DpeInstance::new(
             &mut env,
-            Support::AUTO_INIT | Support::INTERNAL_INFO | Support::X509,
+            Support::INTERNAL_INFO | Support::X509 | Support::AUTO_INIT | Support::ROTATE_CONTEXT,
         )
         .unwrap();
 
-        DeriveChildCmd {
+        let Response::RotateCtx(NewHandleResp { handle, .. }) = RotateCtxCmd {
             handle: ContextHandle::default(),
+            flags: RotateCtxFlags::empty(),
+        }
+        .execute(&mut dpe, &mut env, TEST_LOCALITIES[0])
+        .unwrap() else {
+            panic!("Rotate Context failed");
+        };
```

Comment:
nit (throughout this test): just calling `unwrap` with no else will give more information (like the error code). So I think that's prefferable.

Reply:
Done

Reply:
Oh, ok. In that case, it might be good to do something like (this probably doesn't compile, but close enough)

```rust
match execute() {
  Ok(RotateContextResp(resp)) => something,
  Ok(_) => panic!("Invalid enum variant"),
  Err(e) => panic!("{}", e),
}
```

Reply:
Done